website/docs/r/iot_topic_rule.html.markdown

```diff
@@ -76,6 +88,7 @@ EOF
 * `enabled` - (Required) Specifies whether the rule is enabled.
 * `sql` - (Required) The SQL statement used to query the topic. For more information, see AWS IoT SQL Reference (http://docs.aws.amazon.com/iot/latest/developerguide/iot-rules.html#aws-iot-sql-reference) in the AWS IoT Developer Guide.
 * `sql_version` - (Required) The version of the SQL rules engine to use when evaluating the rule.
+* `error_action` - (Optional) The error action to be associated with the rule.
```

Comment:
Since we are adding a new configuration block, we should include all the details of the nested configuration blocks (or at least reference them). 👍 

```
* `error_action` - (Optional) Configuration block with error action to be associated with the rule. See the documentation for `cloudwatch_alarm`, `dynamodb`, `dynamodbv2`, ....... configuration blocks for further configuration details.
```